gridcomps/History/MAPL_HistoryGridComp.F90

```diff
@@ -1807,6 +1823,8 @@ subroutine Initialize ( gc, import, dumexport, clock, rc )
                   call MAPL_LocStreamCreate(IntState%Regrid(n)%PTR%locIn, &
                        layout, FILENAME=IntState%Regrid(n)%PTR%TILEFILE, &
                        NAME='history_in', MASK=(/MAPL_Ocean/), grid=grid_in, _RC)
+
+                  write(6,*) 'test if (found) from L1747 '
```

Comment:
I think you might want to remove this debug print

Reply:
Thank you! Oops, I will check more of these trailing debug info.

Reply:
Line deleted, thanks!